src/js/filemanager.js

```diff
@@ -2204,7 +2192,19 @@ $.richFilemanagerPlugin = function(element, pluginOptions)
                 }
             }
         };
-
+        var UpdateModel = function() {
+            this.sendUpdate = function () {
+                var interval = config.update.interval
+                var currRequest = new Date()
+                if(this.lastRequest && currRequest - this.lastRequest < interval) {
+                    console.log('minimal interval between updates is ' + interval + 'ms')
```

Comment:
rozumiem, że user nic nie widzi i nie musimy mu pokazywać komunikatu?

Reply:
Faktycznie masz racje @meloniasty, że tutaj używamy ten interval. 

BTW, fajnie byłoby, żeby user wiedział, że nie może wywołać bo, ale to pewnie trzeba by deaktywować przycisk w FinCRM na określony właśnie czas po kliknięciu.